server/app/assets/javascripts/admin_program_block_edit.ts

```diff
@@ -0,0 +1,48 @@
+import {addEventListenerToElements} from './util'
+import {init as mainInit} from './main'
```

Comment:
qq- won't it create circular dependency? init calling AdminProgramBlockEdit  and AdminProgramBlockEdit calling init?